pkg/apis/rbac/helpers.go

```diff
@@ -254,3 +254,63 @@ func (r *ClusterRoleBindingBuilder) Binding() (ClusterRoleBinding, error) {
 
 	return r.ClusterRoleBinding, nil
 }
+
+// +k8s:deepcopy-gen=false
```

Comment:
> I'm not a huge fan of having helpers in the internal types api packages. @deads2k

RBAC really demonstrates the issue with using external versions.  The RBAC authorizer doesn't care which version of the API you use, but you run into crazy problems with caches, informers, conversions, and helpers if you don't use the internal types.

I suspect that if you saw the code duplication involved in having v1beta1 and v1alpha1 support, you'd be displeased.